db.go

```diff
@@ -2050,11 +2088,17 @@ func updateObjects(ctx context.Context, db *DB, exec Executor, list []interface{
 //		defer finishTracer()
```

Comment:
- [ ] update comment's usage example to include `errPtr *error`

Reply:
Good catch, thanks.